### PR TITLE
Add terpene module

### PR DESCRIPTION
**Pull request by lisavader**
Add terpene analysis module

### CODE REVIEW

**Comment by kblin** [comment]
There's dead code in the integration tests stopping me from merging this. Can you either use that where it should be used or remove it?

**Comment by kblin** [comment]
there's also quite a number of additional pylint issues

File: antismash/modules/terpene/data/all_profiles.hmm
Comment (SJShaw) [inline antismash/modules/terpene/data/all_profiles.hmm:1]: This should be in `.gitignore`, since it's already being rebuilt when necessary and just duplicates the other profiles.

File: antismash/modules/terpene/html_output.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+def format_subtype(prediction: DomainPrediction) -> str:
+    if not prediction.subtypes:
+        if _hmm_properties[prediction.type].subtypes:
+            return "unknown"
+        return "none"
+    descriptions = []
+    for subtype in prediction.subtypes:
+        try:
+            short_description = "".join(_hmm_properties[subtype].description.split("; ")[1:])
+        except:
```
Comment (SJShaw) [inline antismash/modules/terpene/html_output.py:30]: Bare `except` is dangerous, since it can also swallow syntax errors.

File: antismash/main.py
Comment (SJShaw) [inline antismash/main.py:1]: Don't change unrelated files in a single commit. Do it in a separate commit, if at all.

File: antismash/outputs/html/css/style.scss [hunk 2]
```diff
@@ -1712,7 +1712,7 @@ ul.dropdown-options {
         * th, * td {
             padding: .8em 1em;
         }
-        thead tr v{
```
Comment (SJShaw) [inline antismash/outputs/html/css/style.scss:1715]: Don't include unrelated changes in the same commit. This one in particular might have follow-on effects and it would be nice to know in the history when and why it was changed.

**Review by SJShaw — CHANGES_REQUESTED** [review]
There's some unsafe pieces that need fixing, and a whole lot of linting and style issues.

Make sure you run `pylint` on the files you're changing, along with flake8`.

**Review by SJShaw — APPROVED** [review]
